muhammet-tarakci-comma-based.csl

```diff
@@ -0,0 +1,1300 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" and="symbol" delimiter-precedes-et-al="never" delimiter-precedes-last="never" page-range-format="chicago">
```

Comment:
you can add a default locale in line 2 with 'default-locale="es-ES"' . Change the country code accordingly.